test-results.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@Kyrus2504 can you remove this file please, you don't need to push this on to GitHub, the xml file is for our use case only for debugging purposes.

Reply:
Sorry! Should be fixed now :)